Source/Core/Editor/Unity/PreBuildLinkXMLCreator.cs

```diff
@@ -0,0 +1,48 @@
+using System.IO;
+using UnityEditor.Build;
+using UnityEditor.Build.Reporting;
+
+/// <summary>
+/// Generates a link.xml file before the build process to preserve all Behaviors and Conditions from VR Builder add-ons.
+/// </summary>
+public class PreBuildLinkXMLCreator : IPreprocessBuildWithReport
+{
+  public int callbackOrder => 3;
+  public const string ResourceFolder = "Assets/MindPort/VR Builder/Resources";
```

Comment:
The resources folder is a special Unity folder. I wouldn't put in there stuff that we don't need to load with the Resources.Load() command. What about placing this file just in Assets/MindPort/VR Builder? Not as tidy, but it's the kind of file which can stay at the root anyway.

Reply:
I agree

Reply:
I agree, will do.